### ISSUE TITLE
No pip-requirements.txt

### ISSUE THREAD

**Issue by JMAlego** [HackSoc/hacksoc.org#189]
` init-venv.sh ` tries to install pip requirements from the file `pip-requirements.txt` which doesn't exist.

**Comment by JMAlego** [comment]
Did you have a local one @LukeMoll that wasn't committed?

**Comment by LukeMoll** [comment]
No, I think it's leftover from before the hacksoc_org package was created and dependencies moved to setup.cfg. It should probably be deleted so the installation experience is uniform across platforms - I don't think the docs even refer to it still.

**Comment by JMAlego** [comment]
> No, I think it's leftover from before the hacksoc_org package was created and dependencies moved to setup.cfg. It should probably be deleted so the installation experience is uniform across platforms - I don't think the docs even refer to it still.

When you say deleted do you mean the file or the line in the file?

**Comment by LukeMoll** [comment]
Sorry, I meant that init-venv.sh could probably be deleted. Unless people
think that it would significantly lower barrier of entry vs copy-pasting
the commands in the README, I think getting rid of it will make it easier
to troubleshoot newcomers' problems. If we were to keep it then someone
should write an equivalent file for Windows.
